tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup.yaml

```diff
@@ -18,7 +18,7 @@ spec:
           katacontainers.io/kata-runtime: cleanup
       containers:
       - name: kube-kata-cleanup
-        image: quay.io/kata-containers/kata-deploy:latest
+        image: quay.io/kata-containers/kata-deploy:2.4.0-rc0
```

Comment:
Wait, why are we removing the stable & latest deployments in favour of current rc? I also think this would need a doc update.

Reply:
tbh, it seems a bit crazy :)

There's a follow in automated PR for setting back to latest after the tag is completed: https://github.com/kata-containers/kata-containers/pull/3846

Reply:
@Jakob-Naucke, @egernst, please, take a look at https://github.com/kata-containers/kata-containers/pull/3089